package.json

```diff
@@ -3,28 +3,30 @@
   "version": "6.5.0",
   "description": "An ESLint plugin for linting ESLint plugins",
   "author": "Teddy Katz",
-  "main": "./lib/index.js",
+  "main": "./dist/index.js",
   "type": "module",
   "exports": {
-    ".": "./lib/index.js",
+    ".": "./dist/index.js",
     "./package.json": "./package.json"
   },
   "license": "MIT",
   "scripts": {
+    "build": "tsup",
     "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
     "lint:docs": "markdownlint \"**/*.md\"",
-    "lint:eslint-docs": "npm-run-all \"update:eslint-docs -- --check\"",
+    "lint:eslint-docs": "npm-run-all -s build \"update:eslint-docs -- --check\"",
     "lint:js": "eslint --cache --ignore-pattern \"**/*.md\" .",
     "lint:js-docs": "eslint --no-inline-config \"**/*.md\"",
     "lint:package-json": "npmPkgJsonLint .",
     "release": "release-it",
     "test": "vitest run --coverage",
-    "test:remote": "eslint-remote-tester -c ./eslint-remote-tester.config.ts",
+    "test:remote": "eslint-remote-tester",
```

Comment:
Does it detect the TS config file automatically now?

Reply:
It does!